snapd-glib/requests/snapd-request-async.c

```diff
@@ -137,6 +139,26 @@ _snapd_request_async_report_progress (SnapdRequestAsync *self, SnapdClient *clie
     }
 }
 
+SnapdGetChange *
+_snapd_request_async_make_get_change_request (SnapdRequestAsync *self)
+{
+    SnapdRequestAsyncPrivate *priv = snapd_request_async_get_instance_private (self);
+    SnapdGetChange *request = _snapd_get_change_new (priv->change_id, NULL, NULL, NULL);
+
+    _snapd_get_change_set_api_path (request, priv->change_api_path);
+    return request;
+}
+
+SnapdPostChange *
+_snapd_request_async_make_post_change_request (SnapdRequestAsync *self)
+{
+    SnapdRequestAsyncPrivate *priv = snapd_request_async_get_instance_private (self);
+    SnapdPostChange *request = _snapd_post_change_new (priv->change_id, "abort", NULL, NULL, NULL);
```

Comment:
I think the `"abort"` here should be a parameter as should the cancellable param etc. But in this case it's not a blocker as this is internal API.